pkg/dns/dns.go

```diff
@@ -225,6 +225,8 @@ func (kd *KubeDNS) setEndpointsStore() {
 				// TODO: Avoid unwanted updates.
 				kd.handleEndpointAdd(newObj)
 			},
+			// No DeleteFunc for EndpointsStore because endpoint object would not be deleted
+			// until corresponding service is deleted.
```

Comment:
Would it be clearer to say: .. will be deleted when the corresponding service is deleted?